tests/foreman/sys/test_foreman_rake.py

```diff
@@ -0,0 +1,47 @@
+"""Test class foreman_rake
+
+:Requirement: Other
+
+:CaseAutomation: Automated
+
+:CaseLevel: System
+
+:CaseComponent: contentManagement
```

Comment:
This shouldn't be contentManagement, as only some rake commands will fit under that. Perhaps using other or removing this field altogether.

Reply:
@JacobCallahan As you said we need new metadata from which I understand, this is adding our testing scope. Do we really want to add our testing scope?

Reply:
Setting `:CaseComponent:` to `Infrastructure` seems a good fit for me, as this component is for "startup, shutdown, task queue any **server side command utilities**".

Historically, bugs in `rake` were filled against "Installer" (21 bugs), "Content Management" (15 bugs), "Infrastructure" (10 bugs), "Other" (10 bugs), "Tasks Plugin" (10 bugs) and many others.

One benefit of leaving it at `ContentManagement` is that Lai is primary owner of that component.

Reply:
@mirekdlugosz Infrastructure may not be the correct component, as I believe it truly refers to areas like Capsules, Domains, Subnets, etc..

Reply:
I've pushed the latest changes and left caseComponent as contentmanagement since we didn't reach a consensus.